packages/react-layouts/src/Modal/index.test.js

```diff
@@ -126,13 +128,69 @@ describe('Modal', () => {
   });
   it('renders a vertically centered modal', () => {
     const wrapper = mount(
-      <Modal centerVertically contentLabel="Foobar" isOpen>
+      <Modal
+        theme={{ current: 'dark', colors: { black: '#000' } }}
+        centerVertically
+        contentLabel="Foobar"
+        isOpen
+        importance={'warning'}
+      >
         <span>Hello, World!</span>
       </Modal>
     );
     expect(wrapper).toMatchSnapshot();
   });
 
+  it('renders Header, HeaderIcon, ActionBar with different themes', () => {
+    const ActionDarkHeader = mount(
+      <Header
+        theme={{ current: 'dark', colors: { black: '#000' } }}
+        title={'Hello world'}
+        icon={<i />}
+        importance={'action'}
+      />
+    );
+    expect(ActionDarkHeader).toMatchSnapshot();
```

Comment:
These could become more concise if you used `toHaveStyleRule` rather than `toMatchSnapshot`